src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -102,6 +103,9 @@ export function fetchFutureAppointments({ includeRequests = true } = {}) {
     const useFeSourceOfTruth = selectFeatureFeSourceOfTruth(state);
     const useFeSourceOfTruthCC = selectFeatureFeSourceOfTruthCC(state);
     const useFeSourceOfTruthVA = selectFeatureFeSourceOfTruthVA(state);
+    const useFeSourceOfTruthModaility = selectFeatureFeSourceOfTruthModality(
```

Comment:
```suggestion
    const useFeSourceOfTruthModality = selectFeatureFeSourceOfTruthModality(
```

nit: spelling error, this appears in a lot of places.

Reply:
Done